Il2CppInspector.Common/Outputs/ScriptResources/shared-main.py

```diff
@@ -96,9 +96,16 @@ def ProcessJSON(jsonData):
 		DefineILMethodInfo(d)
 
 	# Function boundaries
-	print('Processing function boundaries')
```

Comment:
You accidentally deleted this line :)

Reply:
sorry for that,I direct edit on github web.:)